cpu/esp32/periph/can.c

```diff
@@ -259,16 +259,26 @@ static int _esp_can_send(candev_t *candev, const struct can_frame *frame)
     esp_frame.rtr = (frame->can_id & CAN_RTR_FLAG);
     esp_frame.eff = (frame->can_id & CAN_EFF_FLAG);
 
+    /* esp_frame is a union, a concept that cppcheck does not understand. For
+     * this reason cppcheck incorrectly detects values of assignments to union
+     * members as unused (unreadVariable). But their memory is accesses via
+     * union aliases */
```

Comment:
Note: I was unable to reproduce this locally. The most recent `cppcheck` does understand `union`s and is aware that dead writes to `volatile` variables are usually intentional, but it needs full access to all headers. I think here it is missing the include path to `stdint.h`. Or maybe the CI is just using an outdated version of `cppcheck`. I will adapt the comment a bit to be less snarky about `cppcheck`, as missing include paths (if this is indeed the issue) is not really the fault of the tool.